src/templates/default.json

```diff
@@ -341,28 +336,27 @@
           "leftIcon": "",
           "size": "md",
           "key": "submit",
+          "tableView": false,
           "label": "Submit",
-          "input": true,
-          "type": "button"
+          "input": true
         }
       ]
     },
-    "adminLogin": {
-      "title": "Admin Login",
+    "guardianRegister": {
```

Comment:
Same question here: Do we need a guardian register form if we're going to be registering guardians via the backend API?

Reply:
I believe the backend api uses the form itself to operate for this.